buzzmobile/launch/simulation.launch

```diff
@@ -9,7 +9,8 @@
         <include file="$(find buzzmobile)/launch/includes/mapping.launch"/>
         <include file="$(find buzzmobile)/launch/includes/steering.launch"/>
         <include file="$(find buzzmobile)/launch/includes/lidar.launch"/>
-        <include file="$(find buzzmobile)/launch/includes/controller.launch"/>
+        <node pkg="buzzmobile" name="car_pose_mux" type="car_pose_mux.py"/>
+        <!--<include file="$(find buzzmobile)/launch/includes/controller.launch"/>-->
```

Comment:
I made a comment but idk why its not showing up. Id rather have a redundant controller node started than be launching nodes in two different ways in this file.

Others can argue otherwise, but in either case, this change doesnt belong in this PR (unless this is required to make sim work). If you feel strongly abt it then make a new PR pls